src/dex-ui/services/contractsUAT.json

```diff
@@ -28,12 +28,12 @@
   },
   {
     "name": "ftdaofactory",
-    "id": "0.0.89664",
-    "hash": "e392e12b48596352402bbc259151f400",
-    "address": "0x0000000000000000000000000000000000015e40",
-    "timestamp": "2023-07-28T06:23:29.132Z",
-    "transparentProxyId": "0.0.89731",
-    "transparentProxyAddress": "0x0000000000000000000000000000000000015e83"
+    "id": "0.0.465231",
+    "hash": "0dffef715e71711c9ebc34c241ff9cce",
+    "address": "0x000000000000000000000000000000000007194f",
+    "timestamp": "2023-08-09T12:42:00.782Z",
+    "transparentProxyId": "0.0.465235",
+    "transparentProxyAddress": "0x0000000000000000000000000000000000071953"
   },
   {
     "name": "multisigdaofactory",
```

Comment:
<img width="1690" alt="Screenshot 2023-08-10 at 5 16 59 PM" src="https://github.com/hashgraph/hedera-accelerator-defi-dex-ui/assets/89677167/a28bc980-3fcd-40c2-9809-956ffec173b9">


we also need to include the asset tab to see which and how many tokens are associated `DAO` same as in `MultiSig`.

Reply:
Assets Changes Will be covered with Next ticket: https://rally1.rallydev.com/#/641170237735d/iterationstatus?detail=%2Ftask%2F719287236301